### PR TITLE
Add buttons

### PR DESCRIPTION
**Pull request by joaojusto**
Why:

* We need a reusable component for our buttons
* We need at least two different color themes for the buttons;

This change addresses the need by:

* Adding a new component named `Button`;
* Adding `<Button />` to `kitchensink` showing the two themes;

Screenshot 2020-04-11 at 12 35 51

### CODE REVIEW

**Review by jfranciscosousa — APPROVED** [review]

File: src/components/Button/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import PropTypes from "prop-types";
+
+import styles from "./index.module.css"
+
+const THEMES_VALUES = [
+  styles.redTheme,
+  styles.neonCarrotTheme,
+]
+
+export const THEMES = {
```
Comment (jfranciscosousa) [inline src/components/Button/index.js:8]: How will this work if we have a bunch of components with a bunch of different themes?

Comment (joaojusto) [reply src/components/Button/index.js:8]: That's a great question, I don't know how to answer that yet. Maybe the themes can be abstracted, as they should be the same for all?

**Comment by joaojusto** [comment]
@jfranciscosousa updated with variables instead of hardcoded values

File: src/components/Button/index.module.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  display: inline-block;
+
+  padding: $size-26 $size-36;
+  border-radius: $size-26;
+
+  border: none;
+
+  font-size: $size-26;
+  line-height: $size-36;
+  font-family: "Gil Sans";
```
Comment (jfranciscosousa) [inline src/components/Button/index.module.css]: You can cut this off, we set the `font-family` globally elsewhere

**Review by jfranciscosousa — APPROVED** [review]

File: src/components/Button/index.module.css (hunk 2 above)
Comment (joaojusto) [reply src/components/Button/index.module.css]: Done 👍

**Pull request merged**